internal/platform/web/validation.go

```diff
@@ -35,7 +35,11 @@ func (err InvalidError) Error() string {
 // Unmarshal decodes the input to the struct type and checks the
 // fields to verify the value is in a proper state.
 func Unmarshal(r io.Reader, v interface{}) error {
-	if err := json.NewDecoder(r).Decode(v); err != nil {
+
```

Comment:
Please remove this linespace and the linespace above the call to decode.Decode and I will accept the PR.

Thank you for this!

Reply:
I have removed linespace